kytos/core/interface.py

```diff
@@ -451,38 +407,92 @@ def add_tags(self, tags: list[int], tag_type: str = 'vlan') -> bool:
     def make_tags_available(
         self,
         controller,
-        tags: Union[int, list[int]],
+        tags: Union[int, list[int], list[list[int]]],
         tag_type: str = 'vlan',
         use_lock: bool = True,
-    ) -> bool:
-        """Add a specific tag in available_tags.
+        check_order: bool = True,
+    ) -> list[list[int]]:
+        """Add a tags in available_tags.
 
         Args:
             controller: Kytos controller
-            tags: List of tags, there should be 2 items in the list
+            tags: value to be added, multiple types for compatibility:
+                (int): Single tag
+                (list[int]): Single range of tags
+                (list[list[int]]): List of ranges of tags
```

Comment:
If you could further document what `check_order` is for and when to set as False that'd be great for future developers who will use it.

Reply:
Updated docstring